Packs/SymantecManagementCenter/ReleaseNotes/1_0_11.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Symantec Management Center
+- Updated add_policy_content_request() function to prevent additions of duplicate 'url’ entries
```

Comment:
```suggestion
- Updated the **symantec-mc-add-policy-content** command to prevent additions of duplicate 'url’ entries
```
The release notes are for our customers, who are not interested in `functionalities` but only in `commands`.